compiler/control/OMROptions.cpp

```diff
@@ -1487,6 +1489,9 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
      TR::Options::set32BitNumeric, offsetof(OMR::Options, _inlinerVeryColdBorderFrequency), 0, "F%d" },
     { "inlinerVeryColdBorderFrequencyAtCold=", "O<nnn>\tblock frequency threshold for not inlining at cold",
      TR::Options::set32BitNumeric, offsetof(OMR::Options, _inlinerVeryColdBorderFrequencyAtCold), 0, "F%d" },
+    { "inlinerVeryColdBorderFrequencyServer=",
+     "O<nnn>\tblock frequency threshold for inlining in server configurations", TR::Options::set32BitNumeric,
```

Comment:
I find the two option descriptions a little confusing.  Why is `inlinerBorderFrequencyServer` the "block frequency threshold for _not_ inlining" [emphasis, mine], while `inlinerVeryColdBorderFrequencyServer` is "block frequency threshold for inlining"?  Does the first prevent inlining if the block frequency is above the threshold, while the latter permits (forces?) inlining if the block frequency is above the threshold?